Debian-specific patch by Sergei Golovan which replaces current directory
for tclvfs library by a fixed one.

--- tclvfs-1.3-20080503.orig/pkgIndex.tcl.in
+++ tclvfs-1.3-20080503/pkgIndex.tcl.in
@@ -12,19 +12,18 @@
 package require Tcl 8.4
 
 namespace eval ::vfs {}
-variable vfs::dll [file join $dir @PKG_LIB_FILE@]
+variable vfs::dll [file join /usr/lib @PACKAGE_NAME@@PACKAGE_VERSION@ @PKG_LIB_FILE@]
 
-proc loadvfs {dll} {
+proc loadvfs {dir dll} {
     global auto_path
     if {![file exists $dll]} { return }
-    set dir [file dirname $dll]
     if {[lsearch -exact $auto_path $dir] == -1} {
 	lappend auto_path $dir
     }
     load $dll
 }
 
-package ifneeded vfs        @PACKAGE_VERSION@ [list loadvfs $vfs::dll]
+package ifneeded vfs        @PACKAGE_VERSION@ [list loadvfs $dir $vfs::dll]
 
 # Allow optional redirect of VFS_LIBRARY components.  Only necessary
 # for testing, but could be used elsewhere.
